openverse_catalog/dags/providers/provider_workflows.py

```diff
@@ -54,6 +55,9 @@ class ProviderWorkflow:
     doc_md:            string which should be used for the DAG's documentation markdown
     media_types:       list describing the media type(s) that this provider handles
                        (e.g. `["audio"]`, `["image", "audio"]`, etc.)
+    preingestion_task_creator: callable that returns an airflow task or task group to
```

Comment:
And another note here on the post-ingestion task creator too 😄

Reply:
Oops! Added.